CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+This document <https://github.com/gitops-working-group/gitops-working-group/blob/main/CONTRIBUTING.md> defines the contribution process for the GitOps Working Group project and community.
+These guidelines apply to all git repositories in the `gitops-working-group` GitHub org, however each repository may specify additional contribution processes.
+
+## Getting Involved
+
+With the announcement of the GitOps Working Group the founders would like to invite other companies to join the group and contribute to the community and the adoption of GitOps across the cloud native landscape.
+There are a few ways you can get involved:
+
+- Watch or star this repo to see when things change.
+- Attend a [Working Group meeting](./MEETINGS.md).
+- Join `#wg-gitops` on [CNCF Slack](https://slack.cncf.io/) and share how you're using GitOps and any important considerations we should include.
+
+We'll review all open issues and PRs at our regular working group meeting (schedule coming soon).
+
+### Pull Request Tips - Submitting and Reviewing
+
+- Talk to us via GitHub issue, mailing list, or meeting before you start on major
+work. If it’s typos and fixes to docs, please submit. 😀
+- When making changes to your PR, put them into a new commit (instead of rebasing
+  or amending) so that it's easier for reviewers to see just your new changes.
+- We will give you an opportunity to clean up your commits before merging or can
+squash them for you.
+- If a member asks you to "rebase" your PR, they're saying that a lot of docs/code
```

Comment:
Hope the expectation isn't to squash commits, we should be using the Github feature, "squash and merge" unless a PR has multiple 'interesting' commits. :)

Reply:
We probably should change this. This was drawn from elsewhere, and I personally do advocate for exactly what you suggested. The maintainers who merge a PR can decide whether the commits are appropriate to "squash and merge" or perform a merge commit with all the commits in the PR as-is.